cli.js

```diff
@@ -64,11 +65,26 @@ const mds = all
 
 const groupedMds = mds.reduce(groupByPath, []);
 
+// build siteData json
+let siteData = [];
```

Comment:
This pattern you are using here (let variable, and using a map to iterate and build an accumulator) is actually what `reduce` is for!

Also `map` will produce an array with whatever you return (nothing) so it is kind of wasteful. So the `map` should be a `forEach`, but I would really prefer it if you use `reduce` for it.

Ex:

```js
const siteData = mds.reduce((arr, f) => {
  // Do stuff to arr or f...
  return arr
}, [])
```